index.js

```diff
@@ -62,6 +63,8 @@ module.exports = function (opts) {
 				}
 
 				if (stat.isDirectory() || entry.fileName.charAt(entry.fileName.length - 1) === '/') {
+					new Mode(stat).isDirectory(true);
```

Comment:
Oh, really? Can you show us `console.log` result of [this code](https://github.com/kevva/decompress-unzip/issues/6#issue-65966351)?